### PR TITLE
remove sig from create account and burn exec

### PR DESCRIPTION
**Pull request by vaibhavchellani**

### CODE REVIEW

File: test/Reddit.spec.ts [hunk 1]
```diff
@@ -2,400 +2,478 @@ import * as utils from "../scripts/helpers/utils";
...
-            newBalanceRoot,
-            Usage.BurnExecution,
-            { value: ethers.utils.parseEther("32").toString() }
-        );
-
-        assert.equal(newBalanceRoot, await accountStore.getRoot());
-
-    })
-})
+  let wallets: any;
```
Comment (ChihChengLiang) [inline test/Reddit.spec.ts:23]: Could we avoid breaking the style in such a big diff? We could have another PR to unify the style of the whole codebase.

Comment (vaibhavchellani) [reply test/Reddit.spec.ts:23]: Ah! yeah, sorry about that.

**Comment by vaibhavchellani** [comment]
@ChihChengLiang please hold off merging, there are some more changes coming up.

**Comment by ChihChengLiang** [comment]
@vaibhavchellani sure. Please do a `git pull --rebase` to get the latest commits.

**Pull request merged**